nais/prod-fss.yml

```diff
@@ -119,13 +119,13 @@ spec:
     - name: DOKARKIV_SCOPE
       value: api://prod-fss.teamdokumenthandtering.dokarkiv/.default
 
+    - name: NOTAT_ENABLED
+      value: "true"
+
       # Integrasjoner (Kafka)
-      # topic (p:k9-punsj -> c:k9-los-api) for å lage oppgaver i k9-los
     - name: SEND_BREVBESTILLING_TIL_K9_FORMIDLING
       value: "privat-k9-dokumenthendelse"
-      # topic (p:k9-punsj -> c:k9-formidling) for å opprette en dokumentbestilling
     - name: SEND_AKSJONSPUNKTHENDELSE_TIL_K9LOS
       value: "privat-k9punsj-aksjonspunkthendelse-v1"
-
-    - name: NOTAT_ENABLED
-      value: "true"
+    - name: MOTTAK_JOURNALPOST_FRA_FORDEL
```

Comment:
Denne er ikke i bruk enda, jag ska få in den + rapid her i en ny PR så vi har oversikt over alle topics i Nais.yaml.